app/controllers/posts_controller.rb

```diff
@@ -208,6 +208,10 @@ def update
       edit_reason: params[:post][:edit_reason]
     }
 
+    Post.plugin_permitted_update_params.each do |param, _|
```

Comment:
```suggestion
    Post.plugin_permitted_update_params.keys.each do |param|
```
I think this is slightly clearer - you only want the keys so why bother with the `_`?

Reply:
agreed